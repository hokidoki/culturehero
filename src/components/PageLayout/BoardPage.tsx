import type BoardPageProps from './BoardPage.types';
import BoardPageStyle from './BoardPage.styles';

export function BoardPage({ children }: BoardPageProps) {
  return (
    <BoardPageStyle.Layout>
      <BoardPageStyle.Title>Content Lists</BoardPageStyle.Title>
      {children}
    </BoardPageStyle.Layout>
  );
}
