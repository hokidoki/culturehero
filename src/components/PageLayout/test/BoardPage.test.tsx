import { fireEvent, render } from '@render';
import BoardPage from '../BoardPage';

describe('BoardPage', () => {
  it('match snapshot ', () => {
    const utils = render(<BoardPage />);
    expect(utils.container).toMatchSnapshot();
  });
});
