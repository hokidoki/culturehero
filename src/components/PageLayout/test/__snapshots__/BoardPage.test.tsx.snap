// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BoardPage match snapshot  1`] = `
.c0 {
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
  align-items: center;
  width: 100%;
  min-height: 100%;
  background-color: rgb(17,17,20);
  padding: 40px;
}

.c1 {
  font-size: 2.5rem;
  color: white;
  margin-bottom: 60px;
}

<div>
  <div
    class="c0"
  >
    <h1
      class="c1"
    >
      Content Lists
    </h1>
  </div>
</div>
`;
